project/src/main/webapp/education.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8">
+    <meta name="Voter Central Education Page"
+          content="This page is dedicated to educating voters on basic 
+          information about voting in the United States">
```

Comment:
nit. Indent this https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping